astropy/io/ascii/tests/test_connect.py

```diff
@@ -1,4 +1,6 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
+import re
```

Comment:
Why is this added?

Reply:
Apparently, autocomplete went wild when I made the change here. This was a mistake.